packages/nodes-base/nodes/Notion/GenericFunctions.ts

```diff
@@ -25,6 +26,22 @@ import moment from 'moment-timezone';
 
 import { validate as uuidValidate } from 'uuid';
 
+function uuidValidateWithoutDashes(this: IExecuteFunctions, value: string) {
+	if (!value || typeof value !== 'string') return false;
+	if (uuidValidate(value)) return true;
+	if (value.length == 32) {
+		//prettier-ignore
+		const strWithDashes = `${value.slice(0, 8)}-${value.slice(8, 12)}-${value.slice(12, 16)}-${value.slice(16, 20)}-${value.slice(20)}`;
+		if (uuidValidate(strWithDashes)) return true;
+	}
+	throw new NodeOperationError(
+		this.getNode(),
+		`The relation id "${value}" is not a valid uuid with optional dashes.`,
+	);
```

Comment:
throwing error here, instead returning false, could be breaking change for some workflows, in case of reported issue it would break user's workflow in case when relations are optional